controllers/stats/index.js

```diff
@@ -0,0 +1,23 @@
+import db from '../../db/models';
+
+export default {
+  update: async (req, res) => {
+    try {
+      const { params: { slug } } = req;
+      const article = await db.Article.findOne({
+        where: { slug }
+      });
+
+      await article.increment(['reads'], { by: 1 });
+      return res.status(200).send({
+        message: 'Article view successfully Incremented',
+      });
+    } catch (e) {
+      /* istanbul ignore next */
+      return res.status(500).send({
+        message: 'Something went wrong',
```

Comment:
Good job @caleb-42, I like the way you structured your **controller** to follow the convention as agreed by the team. But, one correction is needed in the catch block on line 18, we agreed as a team not to return `message: 'Something went wrong`. instead `error: Something went wrong`. Also `e.message should be removed. Thank you

Reply:
I forgot to follow the convention the team agreed on, will change that now